protocols/index/reports/SECURITY.md

```diff
@@ -0,0 +1,151 @@
+
+## Structure
+
+Index is comprised of a single contract.
+
+[@airswap/index/contracts/Index.sol](../contracts/Index.sol) @ [2518c58bec77b8b7d7791d225de9f22716294e77](https://github.com/airswap/airswap-protocols/commit/2518c58bec77b8b7d7791d225de9f22716294e77)
+
+Deployment of Index.sol was performed in the `@airswap/index` package in the `airswap-protocols` monorepo.
+
+## Dependencies
+
+[Open Zeppelin v2.0 Security Audit](https://drive.google.com/file/d/1gWUV0qz3n52VEUwoT-VlYmscPxxo9xhc/view)
+
+_Externally Audited Files from the OpenZeppelin library (v2.2)_
+
+```
+Ownable.sol
+```
+
+_Externally Audited files from OpenZeppelin library used solely for tests_
+
+```
+ERC20Mintable.sol
+```
+
+## Contracts
+
+```
+Index.sol
+** Ownable.sol
+```
+
+_\*\* OpenZeppelin contract_
+
+#### Public and external functions (non-getter functions)
+
+| Function     | Source      | Visibility   | Params                                                             | Payable |
+| :-------     | :---------- | :----------- | :----------------------------------------------------------------- | :------ |
+| setLocator   | Index.sol   | external     | `address _user, uint256 _score, bytes32 _data`                     | no      |
+| unsetLocator | Index.sol   | external     | `address _user`                                                    | no      |
+| fetchLocators| Index.sol   | external view| `uint256 _count`                                                   | no      |
+
+## Invariants
+
+#### 1. No ether is ever held by the contract address.
+
+- No functions are payable and thus stuck ether due to improper function calls is not possible.
+
+- Although self-destruct can forcefully send money to any non-payable contract; this is out of scope of review, and because this.balance is not used in the code, cannot lead to security issues.
+
+- **This invariant currently holds as-is.**
+
+#### 2. The linked list of Locators may only be modified by the contract owner.
+
+- The locatorsLinkedList is modified in 3 places in the contract. In all places, there is a modifier onlyOwner at the beginning:
+    - First, the constructor. The caller of the constructor of the contract becomes the contract’s owner by definition of OpenZeppelin’s Ownable contract.
+    - Second in unsetLocator. This function is marked as onlyOwner, meaning that it can only be called successfully by the contract's owner.
+    - Third in setLocator. This function is marked as onlyOwner, meaning that it can only be called successfully by the contract's owner.
+- **This invariant currently holds as-is.**
+
+
+#### 3. Linked list ordering by score must be preserved at all times (Highest -> Lowest).
+
+a. Inserting Locators preserves the ordering of the list.
```

Comment:
this will be modified/updated with the fixing of the index.

Reply:
we're not fixing it, it's good.